### DISCUSSION TITLE
Issue sending inputs to the game

### DISCUSSION THREAD

**Discussion by drcorraleso** [tanjeffreyz/auto-maple#63] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79153026/167767531-4ce814a5-f6fa-48cc-85dc-826ffeabcfee.png)

Got about this far with the bot after troubleshooting several issues mostly related to paths, any clue what to do next? I'm not really sure what this error is saying.

**Comment by tanjeffreyz** [comment, marked as answer]
Ah really sorry, that was me being very careless. The [newest release](https://github.com/tanjeffreyz/auto-maple/releases/tag/v2.1.2) should have that fixed!
